bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/niri/NiriForm.java

```diff
@@ -2,53 +2,51 @@
 
```

Comment:
Nice cleanup of this messy class, with the elimination of unnecessary stored variables and the action listener redundancy.